tsdat/config/variables/variable_attributes.py

```diff
@@ -214,6 +214,7 @@ class VariableAttributes(AttributeModel):
         ),
     )
 
+    # TODO: Seems like a static method here, should refactor into as such.
```

Comment:
Agreed

Reply:
Resolved in 207ebfc50299bc5938808f6e72df6af65adcebd4.

Reply:
Sorry, I thought you meant extracting the method to a separate function. The validator should be a classmethod since that's a convention for the pydantic version we're on:

https://docs.pydantic.dev/1.10/usage/validators/

Reply:
Ooh gotcha. I haven't used those validators yet. New tool to learn :-) Lemme give it a read over and revise the code again.

Reply:
Resolved in 020b9f76ab373500197fcf5bf03008c8b8f89e32.